src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 # DataSource settings: set here your own configurations for the database
 # connection.
 spring.datasource.url = jdbc:mysql://localhost:3306/dyntree?useSSL=false
-spring.datasource.name=dyntree
+spring.datasource.name = dyntree
 spring.datasource.username = root
 spring.datasource.password = 10!RootRoot
```

Comment:
don't store credentials (well, at least staging/production ones. Some weak local like `password=password` is fine I guess) in repo :-)

Reply:
I know, my laziness here ;). Should I create additional .properties file only for credentials and .gitignore it?

Reply:
sth like that, I usually do a secret.properties file and as you've mentioned - add it to .gitignore